tools/update-readme.js

```diff
@@ -84,20 +95,61 @@ function formatTeamMembers(members) {
  * @returns {string} The HTML for the readme.
  */
 function formatSponsors(sponsors) {
-    const nonEmptySponsors = Object.keys(sponsors).filter(tier => sponsors[tier].length > 0);
+    const nonEmptySponsors = Object.keys(sponsors).filter(
+        tier => sponsors[tier].length);
```

Comment:
```suggestion
        tier => sponsors[tier].length
    );
```

To fix the lint error.

Reply:
ty